### DISCUSSION TITLE
List of pharmacophores' modeling and screening software

### DISCUSSION THREAD

**Discussion by dprada** [uibcdf/Study_Group_Pharmacophores#17] [General]
Let's try to compile, little by little, a list of software to build pharmacophores' models and to work with them for the sake of performing ligand-based or structure-based virtual screenings or de novo design.

To do so, let's add software and features in the file [software_database.md](https://github.com/uibcdf/Study_Group_Pharmacophores/blob/main/software_database.md)
Feel free to add other properties to the table as well as new software entries.

**Comment by Daniel-Ibarrola** [comment]
Some of the software I have found so far:

•	FLAP (Fingerprints for ligands and Proteins) – PR
•	LigandScout – PR
•	MOE (Molecular Operating Environment) – PR, MD 
•	PHASE – PR
•	Align-it – FR
•	Forge - PR
•      Dynophores* - OS, FR, MD, PY

Most of them are not free, and apparently they do not incorporate molecular dynamics. 

 *Software isn't finished

**Reply by dprada** [reply]
I will put all these in the new file.

**Comment by dprada** [comment]
@Daniel-Ibarrola and @LMMV, from this discussion we have to decide in 3 or 4 months if writing a review about open source tools to get pharmacophore models is a good idea.

@Daniel-Ibarrola, this discussion can also be very encouraging to implement an open-source tool. There must be options for those who can not buy a license, or for those who want to implement an improvement or an experimental approach on top of something -not from scratch-. An open-source library, very modular, easily implemented in a workflow or in a "third-party" structure or project. That library can be OpenPharmacophore.

**Reply by Daniel-Ibarrola** [reply]
Yes, an open source tool might be a good idea considering that most software is not free. However, I'm going to keep looking to see if there are other open source and free tools.

**Comment by dprada** [comment]
Let's move the compilation of software to a specific file apart from this discussion. From now on, in a file called 'software_database.md' in this same repository, we will store the list of libraries and software of interest. But let's keep this opened to discuss any issue regarding this list.

**Reply by Daniel-Ibarrola** [reply]
Ok, I will update the file when I find more software.
